code/modules/projectiles/magazines/flamer.dm

```diff
@@ -71,22 +72,21 @@
 			G.update_icon()
 
 /obj/item/ammo_magazine/flamer_tank/afterattack(obj/target, mob/user , flag) //refuel at fueltanks when we run out of ammo.
-	if(!istype(target, /obj/structure/reagent_dispensers/fueltank) && !istype(target, /obj/item/tool/weldpack) && !istype(target, /obj/item/storage/backpack/marine/engineerpack))
-		return ..()
 	if(get_dist(user,target) > 1)
 		return ..()
+	if(!istype(target, /obj/structure/reagent_dispensers/fueltank) && !istype(target, /obj/item/tool/weldpack) && !istype(target, /obj/item/storage/backpack/marine/engineerpack))
+		return ..()
 
-	var/obj/O = target
-	if(!O.reagents || O.reagents.reagent_list.len < 1)
-		to_chat(user, SPAN_WARNING("[O] is empty!"))
+	if(!target.reagents || target.reagents.reagent_list.len < 1)
+		to_chat(user, SPAN_WARNING("\The [target] is empty!"))
```

Comment:
```suggestion
		to_chat(user, SPAN_WARNING("[target] is empty!"))
```
\The is redundant and only needs to be used if you want to force lower or uppercase the - referencing a datum directly will always insert the the. i think it's best to avoid \the so people know that *not* including it won't prevent the insertion of "the" (this should be accomplished by accessing the .name or changing the properness as appropriate) (total "the" count: 7)